fvtest/porttest/makefile

```diff
@@ -54,7 +54,9 @@ OBJECTS := \
   main_function
 
 ifeq (1,$(OMR_PORT_SOCKET_SUPPORT))
-  OBJECTS += omrsockTest
+  ifneq (win,$(OMR_HOST_OS))
```

Comment:
It should have a TODO comment indicating to remove `ifneq (win,$(OMR_HOST_OS))` after the OMRSock API is implemented on Windows.